coverage/io/io.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/sirupsen/logrus"
+)
+
+// CreateMarker produces empty file as marker
+func CreateMarker(dir, fileName string) error {
```

Comment:
Why is this a useful function? It takes as much to call it as it would to just do the call

Reply:
Outstanding